packages/edit-site/src/components/page-pages/style.scss

```diff
@@ -42,4 +15,32 @@
 		// Windows High Contrast mode will show this outline, but not the box-shadow.
 		outline: 2px solid transparent;
 	}
+
+	&.edit-site-page-pages__media-wrapper {
+		width: $grid-unit-50;
+		height: $grid-unit-50;
+		display: block;
+		border-radius: $grid-unit-05;
+		position: relative;
+		background-color: $gray-100;
+		overflow: hidden;
+		flex-grow: 0 !important;
```

Comment:
These styles were just moved, except for this added line. In general I'm not very fond of the styles we're adding because of the need to have a wrapper(`dataviews-view-table__cell-content-wrapper`) with `display:flex`, in table view.